src/octopeer-github/main/DatabaseAdaptable/IKeystrokeEvent.ts

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
I think so, yes, because they are `EventObject`s that need to be accepted by `DatabaseAdaptable`s. I can put them in their own sub-folder if you want, like Mitchell proposed. Maybe even create two sub-folders, one for the `EventObject`s and one for the `DatabaseAdaptable`s?